extensions/json.py

```
@@ -0,0 +1,41 @@
+#
+#   HANDLER
+#
+
+# HANDLER -> LOAD
+from website import *
```

Ruff (0.12.2)

`from website import *` used; unable to detect undefined names

(F403)